pkgs/development/compilers/dmd/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation {
   # Buid and install are based on http://wiki.dlang.org/Building_DMD
   buildPhase = ''
       cd src/dmd
-      make -f posix.mak INSTALL_DIR=$out
+      make -f posix.mak INSTALL_DIR=$out AUTO_BOOTSTRAP=1
```

Comment:
Looking at posix.mk, looks to me like this will download external resources and so breaks sandboxed builds.